dist/svg.js

```diff
@@ -1540,10 +1542,27 @@ SVG.FX = SVG.invent({
       return this
     }
 
+    // set the internal animation pointer at the start position and updates the visualisation
+  , atStart: function() {
+    // Cast boolean to number
+    return this.at(+this.situation.reversed)
+  }
+
+    // set the internal animation pointer at the end position and updates the visualisation
+  , atEnd: function() {
+    // Cast boolean to number
+    return this.at(+!this.situation.reversed)
+  }
+
     // set the internal animation pointer to the specified position and updates the visualisation
   , at: function(pos){
       var durDivSpd = this.situation.duration/this._speed
 
+      // correct position
+      if(pos > 1) pos = 1
+      if(pos < 0) pos = 0
+      if(this.situation.reversed) pos = 1 - pos
```

Comment:
To help you, you can compare it with when the animation is running forward. If I call `at(0.7)` with `reversed:false` it mean that 0.7 of the animation as been done so that is the value that is used to set _start_.